tox.ini

```
@@ -1,5 +1,5 @@
 [tox]
-envlist = py37, py38, py39, py310
+envlist = py39, py310, py311
```

I suspect this should be:

```suggestion
envlist = py39, py310, py311, py312, py313
```

since we support all of those versions currently.